engine/apps/api/views/schedule.py

```diff
@@ -186,8 +199,9 @@ def get_queryset(self, ignore_filtering_by_available_teams=False, annotate=True)
         )
         if not ignore_filtering_by_available_teams:
             queryset = queryset.filter(*self.available_teams_lookup_args).distinct()
-        if not is_short_request or annotate:
-            queryset = self._annotate_queryset(queryset)
+        if not is_short_request:
+            if annotate:
```

Comment:
could this be simplified to `if not is_short_request and annote`?

Reply:
in some cases we need to do `setup_eager_loading`, but don't need to do annotation

Reply:
ignore my comment 😄  I think I glanced over this too quickly the first time